gui/wxpython/tplot/frame.py

```diff
@@ -212,7 +212,7 @@ def _layout(self):
             self.coorval = gselect.CoordinatesSelect(
                 parent=self.controlPanelRaster, giface=self._giface
             )
-        except:
+        except (AttributeError, TypeError, NotImplementedError):
```

Comment:
The purpose of the try block is to catch NotImplementedError, in that case there is no map window and the widget needs to be changed to TextCtrl instead of a widget that allows you to grab coordinates from the map window. So no need for the other errors, the same below.